index.html

```diff
@@ -192,6 +192,22 @@ <h5 class="m-l">HTML</h5>
         </div>
       </div>
     </section>
+    <footer>
+      <div class="mycontainer">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you
+          need built or a project that you needs coding. I'd love to help with it
+        </p>
+      </div>
+      <div class="form-taget">
+      <form action="#" class="form" method="post">
+          <input type="text" name="name" placeholder="Enter your name here" maxlength="30" required>
+          <input type="email" name="email" placeholder="Enter your email addresse here" required>
+          <textarea name="textarea"  id="" cols="30" rows="10" maxlength="500" required>Write your message here</textarea>
```

Comment:
- [ ] kindly use a placeholder in the text area instead of writing it out.
- [ ] please align your submit button to the left like in the original design and kindly add the Button interactions such as hover, active (when it's pressed), as shown in the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&t=nSDtMU2X3yRBcEnw-0) template.
- [ ] your contact section is missing the figma background image with the designs on it. Please fix that
- [ ] also kindly reduce the margin left/right of the input forms and align it with the text above it. this will make it follow the figma design.

yours
![Screenshot (1064)_LI](https://github.com/Ejikeokegbe/Mobile-First-portifolio/assets/65298048/21bab18e-06c3-4a35-ab3c-02805a8f14f1)
figma
![Screenshot (1063)](https://github.com/Ejikeokegbe/Mobile-First-portifolio/assets/65298048/f1b0d0a8-a7ab-49d3-a0b3-6a945a16f4ca)